src/lv_fonts/lv_font_template.c

```diff
@@ -81,10 +81,11 @@ static uint16_t kern_right_gylph_ids[] =
     0x41, 0x342, 0x333
 };
 
-/*Kerning between the respective left and right glyphs*/
+/* Kerning between the respective left and right glyphs
+ * 12.4 format. (rael_value * 16 is stored) */
```

Comment:
Kerning values format is FP4.4 here (sign + 3 bits int + 4 bits fract). + May be worth add comment about scale factor from header.